copilot/fsd-authenticator/manifest.yml

```diff
@@ -62,8 +62,8 @@ variables:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-RedisInstanceURI
 
 secrets:
-  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PUBLIC_KEY_BASE64
-  RSA256_PRIVATE_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PRIVATE_KEY_BASE64
+  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/authenticator_RSA256_PUBLIC_KEY_BASE64
```

Comment:
Shouldn't need to prefix this with `authenticator_` value is shared between apps.

Reply:
Value is already correct and shared, can we revert this change?

Reply:
the RSA256_PUBLIC_KEY_BASE64/RSA256_PRIVATE_KEY_BASE64 are not the same between the _services_ of the application.

Parameter store:

![image](https://github.com/communitiesuk/funding-service-design-authenticator/assets/125875751/e63f946e-f4d7-4f75-870d-166d269a61fc)

As the 'secrets' are stored per _application_ (in this case, preaward), if you have need for multiples of the same named secret (in this case, the public key) then the parameter names need to be unique, or you'd override them

Reply:
After investigation - gonna revert them all to use the same key, for the sake of convenience. At the moment, there is no need to be this specific, and if we need to change how this works, then that should be a seperate block of work. It doesn't change much how this deploys, and saves us some store slots